src/bundle/Resources/public/scss/_content-edit.scss

```diff
@@ -5,40 +5,129 @@
 
 .ez-content-edit-container {
     position: relative;
+    width: 100%;
+    padding-top: calculateRem(55px);
 
     &__close {
-        position: absolute;
-        left: 0;
-        top: 0;
-        font-size: calculateRem(32px);
+        position: fixed;
+        z-index: 3;
+        top: calculateRem(6px);
+        left: calculateRem(16px);
+        width: calculateRem(40px);
+        height: calculateRem(40px);
         color: $ez-black;
-        width: calculateRem(32px);
-        height: calculateRem(32px);
-        margin-left: calculateRem(8px);
-        padding: calculateRem(16px);
-
-        &:before,
-        &:after {
-            content: '';
-            width: calculateRem(32px);
-            height: calculateRem(2px);
-            position: absolute;
-            background-color: $ez-black;
-            top: 50%;
-            left: 0;
-        }
+        text-align: center;
+        line-height: calculateRem(38px);
 
-        &:before {
-            transform: rotate(45deg);
+        &:hover{
+            background: $ez-color-base-light;
+            border-radius: 50%;
+
+            .ez-icon {
+                fill: $ez-white;
+            }
         }
+    }
 
-        &:after {
-            transform: rotate(-45deg);
+    .ez-edit-header {
+        &__content-type-name {
+            position: fixed;
+            z-index: 2;
+            top: 0;
+            left: 0;
+            width: calc(100% - #{calculateRem(80px)});
+            min-height: calculateRem(55px);
+            padding: calculateRem(12px);
+            color: $ez-black;
+            font-size: calculateRem(20px);
+            font-weight: bold;
+            background: $ez-color-base-pale;
         }
 
-        &:hover {
-            text-decoration: none;
+        &__details {
             color: $ez-black;
+            font-size: calculateRem(12px);
+            background: $ez-ground-base-medium;
+            transition: max-height 0.3s;
+            overflow: hidden;
+
+            .ez-details-items {
+                padding: 0 calculateRem(48px);
+                max-height: calculateRem(32px);
+                transition: max-height 0.3s;
+
+                &__pill {
+                    display: inline-block;
+                    margin: calculateRem(2px) calculateRem(3px);
+                    border-radius: calculateRem(4px);
+
+                    &--gray {
```

Comment:
We should not use colors in CSS classes and for me it's more natural to name it by the background not font color.
This I would name `--light`